src/interface.js

```diff
@@ -0,0 +1,394 @@
+    let f10roll1 = $('#f10roll1').val();
+    let f10roll2 = $('#f10roll2').val();
+    let f10roll3 = $('#f10roll3').val();
+    if (f10roll3 == '') {
+      alert('Please enter a value for frame 10, roll 3');
+    }
+    if (f10roll1 == 10) {
+      if (f10roll2 != 10) {
+        if (total(f10roll2, f10roll3) == true) {
+          alert('The sum of your final two rolls cannot exceed 10');
+          $('#f10roll2').val('1');
+          $('#f10roll3').val('1');
+        }
+      }
+    }
+    game.roll(parseInt(f10roll1), parseInt(f10roll2), parseInt(f10roll3));
+    updateScore();
+  });
+  $( '#go' ).click(function() {
+    $('header').hide();
+    $('#l1r1').hide();
+    $('#l1r2').hide();
+    $('#f1roll1').hide();
+    $('#f1roll2').hide();
+    $('#l2r1').hide();
+    $('#l2r2').hide();
+    $('#f2roll1').hide();
+    $('#f2roll2').hide();
+    $('#l3r1').hide();
+    $('#l3r2').hide();
+    $('#f3roll1').hide();
+    $('#f3roll2').hide();
+    $('#l4r2').hide();
+    $('#f4roll1').hide();
+    $('#f4roll2').hide();
+    $('#l5r2').hide();
+    $('#f5roll1').hide();
+    $('#f5roll2').hide();
+    $('#l6r2').hide();
+    $('#f6roll1').hide();
+    $('#f6roll2').hide();
+    $('#l7r1').hide();
+    $('#l7r2').hide();
+    $('#f7roll1').hide();
+    $('#f7roll2').hide();
+    $('#l8r1').hide();
+    $('#l8r2').hide();
+    $('#f8roll1').hide();
+    $('#f8roll2').hide();
+    $('#l9r1').hide();
+    $('#l9r2').hide();
+    $('#f9roll1').hide();
+    $('#f9roll2').hide();
+    $('#l10r1').hide();
+    $('#l10r2').hide();
+    $('#l10r3').hide();
+    $('#f10roll1').hide();
+    $('#f10roll2').hide();
+    $('#f10roll3').hide();
+    $('#f1score').hide();
+    $('#f2score').hide();
+    $('#f3score').hide();
+    $('#f4score').hide();
+    $('#f5score').hide();
+    $('#f6score').hide();
+    $('#f7score').hide();
+    $('#f8score').hide();
+    $('#f9score').hide();
+    $('#f10score').hide();
+    $('#go').hide();
+    $('#reset').show();
+    checkGame();
+    $('#l4r1').text('Your score is:');
+    $('#l5r1').text(game.gameScore.reduce((a, b) => a + b, 0));
+  });
+  $( '#reset' ).click(function() {
+    location.reload();
+  });
+
+  let checkGame = ( ) => {
+    if ((game.gameScore.reduce((a, b) => a + b, 0)) == 300) {
+      $('#l6r1').text('...Perfect game!');
+    }
+    else if ((game.gameScore.reduce((a, b) => a + b, 0)) == 0) {
+      $('#l6r1').text('...Gutter game....');
+    }
+    else {
+      $('#l6r1').hide();
+    }
+  }
+  let updateScore = ( ) => {
+    $('#f1score').text(game.gameScore[0]);
+    $('#f2score').text(game.gameScore[1]);
+    $('#f3score').text(game.gameScore[2]);
+    $('#f4score').text(game.gameScore[3]);
+    $('#f5score').text(game.gameScore[4]);
+    $('#f6score').text(game.gameScore[5]);
+    $('#f7score').text(game.gameScore[6]);
+    $('#f8score').text(game.gameScore[7]);
+    $('#f9score').text(game.gameScore[8]);
+    $('#f10score').text(game.gameScore[9]);
+  }
+});
```

Comment:
I like the use of JQuery and the hideing and text addition, it makes the interface look good. 

A couple of thoughts: 
- there is a lot of duplication here, looks like there is the potential to DRY out. 
- why did you store some of the game logic here? Like the strike and total functions?